<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Screen" name="screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>625</width>
      <height>625</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>621</width>
      <height>621</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chngPixClr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incPixSz">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>660</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decPixSz">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>660</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>231</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="line_ba">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Line_BA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="line_dda">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Line_DDA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="circle_s">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Circle_S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="circle_p">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Circle_P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="circle_b">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Circle_B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ellipse_m">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Ellipse_M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="scan_f">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Scan_Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="flood_f">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
       </property>
       <property name="text">
        <string>Flood_Fill</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="location">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Ubuntu Mono&quot;;
color: rgb(255, 255, 255);
border-width: 2px;
border-style: solid;
border-color: rgb(255, 255, 255);
background-color: rgb(63, 61, 61);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QGraphicsView</extends>
   <header location="global">screen.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
